pkg/podexec/pod_command_executor.go

```diff
@@ -123,6 +123,12 @@ func (e *defaultPodCommandExecutor) ExecutePodCommand(log logrus.FieldLogger, it
 			"hookTimeout":   localHook.Timeout,
 		},
 	)
+
+	if pod.Status.Phase == corev1api.PodSucceeded || pod.Status.Phase == corev1api.PodFailed {
```

Comment:
Could you please add a test case in `pod_command_executor_test.go` to cover this case?

Reply:
ok, i will try do it and add the changlog, ths